mease_elabftw/experiments.py

```diff
@@ -5,6 +5,13 @@
 
 
 def list_experiments(owner=""):
+    """Filter experiments by owner.
```

Comment:
```suggestion
    """Filter experiments by owner.
    
    If no owner is specified, returns all experiments.
```
Otherwise the reader knows the default value for owner is an empty string, but it doesn't tell them what that means they will get back from this function.